src/ITensors.jl

```diff
@@ -132,11 +132,18 @@ include("indexset.jl")
 # ITensor
 #
 include("itensor.jl")
+include("OPS/tensor_algebra.jl")
```

Comment:
I don't think `Ops` is the right location for this, the theme of the `Ops` directory is for files defining the `Op` type (a representation of an operator on sites of a lattice) and lazy operations between `Op`s.

Reply:
I made a new folder called tensor_operations and moved this file there